Clustering/Hdbscan/src/main/resources/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This and the `THIRD_PARTY_LICENSES.txt` files should be symlinks to the ones in the root directory of the repo, rather than copies of those files. That way we don't need to update them everywhere when versions change.